index.js

```diff
@@ -66,7 +65,26 @@ function toggleColor(element) {
         element.style.color = "green"
     }
 }
+// const changeToggle = document.querySelector('h1#header')
+// changeToggle.addEventListener('click', function () {
+//     document.body.classList.toggle('dark-mode')
+// })
+const changeToggle = document.querySelector('h1#header')
+changeToggle.click(function() {
+    ${changeToggle}.toggleClass("black")
+})
+
 
 /***** Deliverable 2 *****/
+const likeButton = document.querySelector('button.like-button')
+const likeCount = document.querySelector('p.likes')
+
+count = 1000
+likeButton.addEventListener('click', function(element){
+    count += 1;
+    likeCount.innerHTML = count + " Likes" ;
+})
+
+
 
 /***** Deliverable 3 *****/
```

Comment:
Here are some steps we could break this problem down to approach it. Review this along with events and see if you're able to accomplish it!

1. Find the sightingForm on the page and save it to a variable
2. add an event listener to the form element, listen for a `submit` event.
3. The submit event needs to be applied to an entire form element, not the submit button.
4. Create the `handleFormSubmission` callback function, which should be triggered when the form submits.
`handleFormSubmission` steps:
1. since we are submitting a form, we need to prevent the default action of the form, which is to make a post request and redirect to a specific URL. We do not want the page to refresh, we want to prevent that action and stay on the current page.
2. save all the data the user typed in from the form into descriptive variables.
3. assemble an object with key value pairs descriptive of the data they contain.
4. Pass in the animal data object to the `renderAnimalSightingPost` function
```